### ISSUE TITLE
I've created todos file but it's not connected

### ISSUE THREAD

**Issue by siddhant660** [siddhant660/To-Do-List#4]
I've created todos.html file but it's not connected with main file meaning it's not showing any todos, pretty confused on backend part, help me with that.

**Comment by joshdev19** [comment]
Yes, because you will need to install mysql or xampp on your machine. 

I'm planning to host our project after we finished it so that you can see it.

btw, Kindly message me here " https://web.facebook.com/jinshin.199 "

Fb, name jinshin, profile pic, anime with a cat
